benchmarks/README.rst

```diff
@@ -0,0 +1,66 @@
+..  -*- rst -*-
+
+================
+SciPy benchmarks
+================
+
+Benchmarking Scipy with Airspeed Velocity.
+
+
+Usage
+-----
+
+Run a benchmark against currently installed Scipy version (don't
+record the result)::
+
+    python runtests.py --bench bench_sparse.Arithmetic
```

Comment:
Yes. I forgot I renamed the files to be shorter.
It might make sense to do some more renaming in the benchmarks still later on,
given that the names are so long.